app/presenters/search_payload_presenter.rb

```diff
@@ -0,0 +1,36 @@
+class SearchPayloadPresenter
+  attr_reader :registerable_edition
+  delegate :slug,
+           :title,
+           :description,
+           :indexable_content,
+           :public_timestamp,
+           :sections,
+           :artefact,
+           :format,
+           to: :registerable_edition
+
+  def initialize(registerable_edition)
+    @registerable_edition = registerable_edition
+  end
+
+  def self.present(registerable_edition)
+    new(registerable_edition).present
+  end
+
+  def present
+    {
+      content_id: artefact.content_id,
+      rendering_app: "publisher",
+      publishing_app: "publisher",
+      format: format.underscore,
+      title: title,
+      description: description,
+      indexable_content: indexable_content,
+      link: "/#{slug}",
+      public_timestamp: public_timestamp,
+      specialist_sectors: registerable_edition.specialist_sectors.compact,
+      sections: sections,
```

Comment:
This is interesting. `sections` are mainstream browse pages, so we should've started indexing these from publishing-api just like `topics` and `specialist_sectors`. 

But we don't: https://github.com/alphagov/rummager/blob/master/lib/indexer/links_lookup.rb

Where did you find `sections` before?